lib/Project.js

```diff
@@ -350,9 +350,10 @@ Project.prototype._resolveExtensions = function () {
 
 /**
  * Executes all the compilation jobs.
+ * @param {?} additionalCompilerOptions these merge onto the default {@link SoyOptions}
  * @return {Promise} A promise of when all compilation jobs have finished
  */
-Project.prototype.compile = function () {
+Project.prototype.compile = function (additionalCompilerOptions) {
```

Comment:
can we make this a setter?

the whole point of the Project class is as a Builder, where properties are set up-front